site/app/templates/AutogradingStatus.twig

```diff
@@ -42,8 +42,17 @@
                     <td>{{ course_name[0] }}</td>
                     <td>{{ course_name[1] }}</td>
                     <td>{{ gradeable }}</td>
-                    <td>{{ types["interactive"] }}</td>
-                    <td>{{ types["regrade"] }}</td>
+					
+                    <td>
+					{% if types["interactive"] != 0 %}
```

Comment:
Use 4 spaces for indentation, not tabs. Need to fix this throughout your changes in twig files.